docs/src/main/tut/streaming.md

```diff
@@ -6,9 +6,7 @@ permalink: /docs/rpc/streaming
 
 # Streaming
 
-In the previous section, we saw that [frees-rpc] allows you to define unary or streaming service methods.
-
-The streaming service methods are the next:
+In the previous section, we saw that [frees-rpc] allows you to define unary services, but additionally supports the next streaming options:
```

Comment:
Suggestion:
In the previous section, we saw that [frees-rpc] allows you to define unary services. It additionally supports the following streaming options: